doc/chrome-webstore-permissions-descriptions.md

```diff
@@ -0,0 +1,29 @@
+Privacy tab in chrome web store:
+-----------------------------------
+
+Permissions:
+
+* `Privacy` -
+Privacy Badger needs access to the privacy API so that it can view the user's current privacy settings such as their webRTCIPHandlingPolicy, then modify or make heuristic decisions on how to bolster their privacy
+
+* `Cookies` -
+Privacy Badger needs access to the cookies API so that it can detect and then prevent if any cookies are being placed on the user's browser to identify and track them.
+
+* `Storage` -
+Privacy Badger needs access to the storage API so that the extension's storage and user's settings will persist beyond a browser session.
+
+* `WebRequest` -
+Privacy Badger needs access to the WebRequest API so that it can view and intercept network requests being made in the browser. If user tracking is taking place via network request headers, Privacy Badger needs to be able to intercept and modify those headers.
+
+* `WebRequestBlocking` -
+Privacy Badger needs access to the WebRequestBlocking API so that it can synchronously view and intercept network requests being made in the browser. If user tracking is taking place via network request headers, Privacy Badger needs to be able to intercept and modify those headers.
```

Comment:
Same as above, why do we need to modify headers and cancel requests? I would broadly list the various specific things that happen inside **synchronous** webRequest listeners.